api/test/gateways/Academy-Benefits/FetchRecord.test.js

```diff
@@ -173,6 +173,17 @@ describe('AcademyBenefitsFetchRecord gateway', () => {
     expect(response.claim_id).toBe(dbRecord.claim_id)
   });
 
+  it("ignore case and whitespace when checking equality", async() => {
+    const record = {...randomRecord1()};
+    const record1 = {...randomRecord1()};
```

Comment:
Same as the comment above here - no need to destructure the new object that is returned by calling the `randomRecord1` function here.

Reply:
I've removed the destructure